boot-script/templates/async.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(app, cb) {
+  
+  cb();
```

Comment:
Good idea. Updating, except for the comment which is too long to fit in 80 chars. I also add the app object comments to `sync.js`.